features/networking/egress-ip.feature

```diff
@@ -178,3 +178,42 @@ Feature: Egress IP related features
     Then the step should succeed
     And the output should not contain "reg0=0x"
     """
+
+  # @author huirwang@redhat.com
+  # @case_id OCP-15992
+  @admin
+  @destructive
+  Scenario: The EgressNetworkPolicy should work well with egressIP
+    Given the valid egress IP is added to the node
+    And I have a project
+    And I have a pod-for-ping in the project
+
+    # Create egressnetworkpolicy
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/egressnetworkpolicy/limit_policy.json |
+      | n | <%= project.name %> |
+    Then the step should succeed
+
+    # add the egress ip to the project
+    Given as admin I successfully merge patch resource "netnamespace/<%= project.name %>" with:
+      | {"egressIPs": ["<%= cb.valid_ip %>"]} |
```

Comment:
Is cb.valid_ip come from "Given the valid egress IP is added to the node"?

Reply:
Yes, it is from that step.